custom_components/govee_ble_hci/govee_advertisement.py

```diff
@@ -89,11 +89,15 @@ def __init__(self, data: bytes):
                 pos += length + 1
 
             if self.check_is_gvh5075_gvh5072():
-                mfg_data_5075 = hex_string(self.mfg_data[3:6]).replace(" ", "")
+                if self.mfg_data[0:3]==b'\x01\x00\x01':
```

Comment:
Instead of piggybacking off of the `self.check_is_gvh5075_gvh5072()`, create a new one `self.check_is_gvh5102()`.   Check the length of the data, if the `mfg_data` begins with this bytes and an appropriate GAP flag.

Reply:
So, the challenge here is that the length and GAP flags are the same :(.  Since that was the only metric being used for determining the model, I tried to stick with what we had.  Alternatively, we could look for length, flag, and the device name (as you mentioned), but then I'd be deviating from _mfg_data_check.